Tests/scripts/slack_notifier.py

```diff
@@ -111,56 +114,137 @@ def options_handler():
     return options
 
 
-def get_failing_unit_tests_file_data():
-    failing_ut_list = None
+def get_artifact_data(artifact_suffix: str) -> Optional[str]:
+    """
+    Retrieves artifact data according to the artifact suffix given.
+    Args:
+        artifact_suffix (str): Artifact suffix.
+
+    Returns:
+        (Optional[str]): data of the artifact as str if exists, None otherwise.
+    """
```

Comment:
Probably be more accurate to describe it as the relative file path to the artifacts directory.